srtp_cipher.go

```diff
@@ -49,3 +49,13 @@ It is *before* the ESRTCP word (Encrypted-flag and SRTCP index).
 
 See https://tools.ietf.org/html/rfc7714 for the full specifications.
 */
+
```

Comment:
Maybe move to test file? Just to make it easier on readers in the future

Reply:
Good idea, will move all test-related stuff to some _test.go file.